src/main/java/com/faltro/houdoku/controller/ReaderController.java

```diff
@@ -137,6 +138,19 @@ public void initialize() {
             double vvalue = imageScrollPane.getVvalue();
             imageScrollPane.setVvalue(vvalue + -deltaY / width);
         });
+        // make pageNumField and totalPagesField grow if the number of pages grows
+        totalPagesField.textProperty().addListener(new ChangeListener<String>() {
+            @Override
+            public void changed(ObservableValue<? extends String> ob, String o, String n) {
+                totalPagesField.setPrefColumnCount(totalPagesField.getText().length());
+            }
+        });
+        pageNumField.textProperty().addListener(new ChangeListener<String>() {
+            @Override
+            public void changed(ObservableValue<? extends String> ob, String o, String n) {
+                pageNumField.setPrefColumnCount(pageNumField.getText().length() +1);
```

Comment:
I don't think this +1 is supposed to be here?

Reply:
No, you're right, it is not supposed to be here. 33acaf3 should fix this.
Thanks !